apps/website/.eslintrc.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   root: true,
   extends: ["custom"],
   parserOptions: {
-    project: ["./tsconfig.json"],
+    project: "./tsconfig.json",
+    tsconfigRootDir: __dirname,
```

Comment:
@qqpann  Noob question. What does this do exactly?

Reply:
When you use parserOptions inside a child directory (not root), you have to specify this or eslint cannot find the tsconfig.json you want.

Reply:
@qqpann  Ah I see. Thanks! 🔥